bin/tat

```diff
@@ -2,7 +2,8 @@
 #
 # Attach or create tmux session named the same as current directory.
 
-session_name="$(basename "$PWD" | tr . -)"
+path_name="$(basename "$PWD" | tr . -)"
+session_name=${1-$path_name}
```

Comment:
@gabebw @graysonwright For future reference, `:-` uses the default when `$x` is either empty or undefined, while `-` only sets the default if undefined.

```
$ X=value eval 'echo ${X:-"default"}'
value
$ eval 'echo ${X:-"default"}'
default
$ X= eval 'echo ${X:-"default"}'
default

$ X=value eval 'echo ${X-"default"}'
value
$ eval 'echo ${X-"default"}'
default
$ X= eval 'echo ${X-"default"}'

```